src/test/java/com/iexec/standalone/replicate/ReplicateSupplyServiceTests.java

```diff
@@ -1169,4 +1169,4 @@ private void mockWorkerCanAcceptMoreWork(final Worker worker) {
         when(workerService.getWorker(worker.getWalletAddress())).thenReturn(Optional.of(worker));
         when(workerService.canAcceptMoreWorks(worker)).thenReturn(true);
     }
-}
+}
```

Comment:
I don't understand this change which adds an issue not present in scheduler code

Reply:
Me neither ... it's probably due to the `rebase`.